src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -514,13 +521,17 @@ public DatasetSummary retrieveSummaryByName(String name) {
             "FROM dataset " + whereSql +
             DaoUtils.orderByClause(sort, direction) + " OFFSET :offset LIMIT :limit";
         params.addValue("offset", offset).addValue("limit", limit);
-        List<DatasetSummary> summaries = jdbcTemplate.query(sql, params, new DatasetSummaryMapper());
+        List<DatasetSummary> summaries = jdbcTemplate.query(sql, params, new DatasetSummaryMapper())
```

Comment:
I agree, but wanted to avoid refactoring SQL queries. I think perf improvements can come once the feature is fully realized. Especially since that will mean we have things to test against to make sure perf improvements don't break the new feature!